src/generic/genvu.tcl

```diff
@@ -22,7 +22,7 @@ proc myerrorproc { id info } {
                     if {[string match {*canceled*} $info]} {
                         #message was eval canceled
                     } else {
-                        puts "Metrics Thread Error: $info"
+			    ;
```

Comment:
Is this oddly spaced semicolon a requirement?  Can we replace it with a descriptive comment like in the "if" block?

Reply:
yes semicolon replace with comment. The section is a placeholder in case additional error messages are required.